python/doc/theory/reliability_sensitivity/sensitivity_hsic.rst

```diff
@@ -0,0 +1,293 @@
+
+HSIC estimators
+~~~~~~~~~~~~~~~
+
+Two alternative estimators exist in order to compute the HSIC value. 
+The first one is a biased, but asymptotically unbiased estimator
+based on V-statistics:
+
+.. math::
+
+    \widehat{\mathrm{HSIC}}(X_i,Y) = \frac{1}{n^2} \mathrm{Tr}(L_i HLH)
+
+where :math:`L_i` and :math:`L` are Gram matrices computed with the respective kernels 
+:math:`L_{i_{j,k}} = \kappa_i(x_i^j,x_i^k)` and :math:`L_{j,k}= \kappa(y^j,y^k)`, 
+while :math:`H` a shift matrix defined as:
+:math:`H_{j,k} = \left(\delta_{j,k} - \frac{1}{n}\right)_{1 \leq j, k \leq n}`.
+
+The second estimator is an unbiased estimator based on U-statistics:
+
+.. math::
+
+	\widehat{\mathrm{HSIC}}(X_i,Y) = \frac{1}{(m)(m-3)} \left[\mathrm{Tr}(\tilde{L}_i \tilde{L}) + \frac{\mathbf{1}^{\top} \tilde{L}_i \mathbf{1}\mathbf{1}^{\top} \tilde{L} \mathbf{1}}{(m-1)(m-2)} - \frac{2}{m-2} \mathbf{1}^{\top} \tilde{L}_i \tilde{L} \mathbf{1}\right]
+
+
+where :math:`\tilde{L}_{i_{j,k}}` and :math:`\tilde{L}_{j,k}` are computed as:
+
+.. math::
+
+    \tilde{L}_{i_{j,k}} = (1-\delta_{j,k}) \kappa\left( Y^{(j)}, Y^{(k)}\right) 
+
+	\tilde{L}_{j,k} = (1-\delta_{j,k}) \kappa_i\left( X_i^{(j)}, X_i^{(k)}\right) 
+
+In order to compare the HSIC values associated to various input variables :math:`X^i`,
+it is common practice to consider a normalized index (bounded between 0 and 1) called R2-HSIC,
+and defined as: 
+
+.. math::
+
+    \widehat{R_{\mathrm{HSIC},i}^2} = \frac{\widehat{\mathrm{HSIC}}(X_i,Y)}{\sqrt{\widehat{\mathrm{HSIC}}(X_i,X_i)~\widehat{\mathrm{HSIC}}(Y,Y)}}
+
+Please note that, differently from the Sobol indices typically used in the context
+of sensitivity analysis, the sum of all normalized R2-HSIC indices for a given set
+of considered input variables does not sum to 1.
+
+Most covariance kernels that can be used in order to compute the Gram matrices of the
+HSIC estimators are characterized by one or several hyper-parameters. No universal
+rule exists allowing to determine the optimal value for these hyper-parameters, however, 
+for some specific kernels empirical rules are proposed. For instance, the squared
+exponential (or Gaussian) kernel can be parameterized as a function of the sample 
+empirical variance. In this case, we obtain :
+
+.. math::
+
+    \kappa_i(x_i^j,x_i^k) = \exp (\theta_i ||x_i^j - x_i^k||^2_2) 
+
+with :math:`\theta_i = 1/\sigma_i^2`, where :math:`\sigma_i^2` is the empirical 
+variance of the sample :math:`X_i`.
+
+Screening with HSIC-based statistical tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The HSIC can also be used in order to perform screening on a set of input variables.
+This can be defined as the the process of identifying the input variables which are 
+significantly influential on the considered output. 
+More specifically, within the framework of HSIC this can be done by relying on 
+statistical hypotheses tests. In practice, we wish to test the the following hypothesis:
```

Comment:
hypothesis tests

Reply:
Done.

Reply:
In this context I decided to use the plural version of hypothesis. However I guess the singular version would work as well.